store/key_value_store_server.cc

```diff
@@ -28,19 +31,65 @@ Status KvstoreServiceImpl::remove(ServerContext* context, const RemoveRequest* r
   }
 }
 
-bool KvstoreServiceImpl::fileExists(const char *fileName)
+bool KvstoreServiceImpl::fileExists(const std::string &fileName)
```

Comment:
This may be redundant as you could just do
`std::ifsteam infile(filename)
if(infile.is_open)` ... in the CreateKeyFile() method.

Reply:
Fixed